### ISSUE TITLE
recording app: N interpretations per locus like in narratives, with time and person stamps

### ISSUE THREAD

**Issue by lbestock** [arch-kiosk/arch-kiosk-office#2396]
We have discussed for ages that interpretation is multiple. Different people, different understandings over time without wanting to erase previous notes. We have also had some concern over where it belongs - is that a recording thing? At the locus level? Isn't it more about relationships between loci, so more on the unit level or even site level?

We may want it elsewhere, but I'm pretty sure we want to have it in the recording app at the interpretation tab of loci, too. I am encountering a need for it today, as I check various Harris Matrix outputs against the recording data. It is already a useful process and would have helped me in writing up this unit. In some cases there are already old notes from multiple times in there, with dates on them; I can add another such (and am doing so). But there is not always a time noted, and no stamp about who _made_ the notes, and those would help me reconstruct the process of interpretation as it has existed to this point. 

In short: an interpretations tab structured much more like the narratives tab would be useful. Create new records, with time and person attached.

**Comment by urapadmin** [comment]
- see also #2880

**Comment by urapadmin** [comment]
we have opted against this as long as we are using FileMaker as recording app.
